tests/unit/lms/services/course_test.py

```diff
@@ -243,6 +278,7 @@ def test_upsert_course_sets_canvas_sections_enabled_based_on_legacy_rows(
             type_=Any(),
             copied_from=None,
         )
+        bulk_upsert.assert_called()
```

Comment:
This test deals with something that's irrelevant in the current LMSCourse model. Just asserting that we are still calling the upsert.

I don't think we should migrate setting/extra as it exists now on Grouping but rather create columns for the information there.

We can do that one bit at a time  once we have the basic structure in place.